CMakeLists.txt

```diff
@@ -650,13 +650,21 @@ if(WIN32)
 	if (MSI_REDISTRIBUTE_UCRT_NONASL)
 		set(UCRT_DIR_NAT "$ENV{WindowsSdkDir}Redist\\ucrt\\DLLs\\$ENV{Platform}")
 		file(TO_CMAKE_PATH "${UCRT_DIR_NAT}" UCRT_DIR)
-		message("Using UCRT from ${UCRT_DIR}")
-		file(GLOB UCRT_DLLS "${UCRT_DIR}/*.dll")
-		file(COPY ${UCRT_DLLS} DESTINATION "${CMAKE_CURRENT_BINARY_DIR}/ucrt")
-		install(DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/ucrt/"
-			DESTINATION bin
-			COMPONENT bin
-		)
+		if (NOT EXISTS "${UCRT_DIR}")
+			set(UCRT_DIR_NAT "$ENV{WindowsSdkDir}Redist\\$ENV{WindowsSDKVersion}ucrt\\DLLs\\$ENV{Platform}")
```

Comment:
does not being in their official/documented directory affect our legal ability to ship these dll-s?

Reply:
We can't distribute these DLLs in any case, the build infrastructure is just there to provide users an easy way to include them for private/internal use.

Reply:
in that case we should update the description of the issue, because the description says "Microsoft allows for the redistribution of these DLLs" while the linked legal ticket, as you said, says otherwise

Reply:
good point, I've updated the issue